Sources/SpeziOpenAI/MessagesView.swift

```diff
@@ -19,6 +19,7 @@ public struct MessagesView: View {
     @Binding var chat: [Chat]
     @Binding var bottomPadding: CGFloat
     let hideSystemMessages: Bool
+    let hideFunctionMessages: Bool
```

Comment:
I noted that we don't forward these properties to the `MessageView`. This is a bug in the current implementation, but we should solve it now as we already touch this system.